demo/index.tsx

```diff
@@ -147,6 +147,10 @@ function App() {
     setIdentityCommitment(idCommitment);
   }, [client, setIdentityCommitment]);
 
+  const createIdentity = useCallback(async () => {
```

Comment:
I think you don't need async/await here and you can use onClick handler without arrow function.